addons/account/models/account_partial_reconcile.py

```diff
@@ -550,6 +550,7 @@ def _create_tax_cash_basis_moves(self):
                     amount_currency = line.currency_id.round(line.amount_currency * partial_values['percentage'])
                     if (
                         caba_treatment == 'tax'
+                        and not self._context.get('skip_tax_residual_fallback')
```

Comment:
What we have to do is put the remaining amount on the last line. We must find a good way to determine which it is